examples/online-shop/service.bal

```diff
@@ -0,0 +1,154 @@
+// Copyright (c) 2021 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/http;
+import ballerinax/oracledb;
+import ballerina/sql;
+
+configurable string USER = "balUser";
+configurable string PASSWORD = "balpass";
+configurable string HOST = "localhost";
+configurable int PORT = 1521;
+configurable string DATABASE = "ORCLCDB.localdomain";
+
+final oracledb:Client dbClient = check new(host = HOST, user = USER, password = PASSWORD, port = PORT,
+    database = DATABASE, connectionPool = {maxOpenConnections: 3, minIdleConnections: 1});
+listener http:Listener onlineShopListener = new (9090);
+
+service /onlineshop on onlineShopListener {
```

Comment:
```suggestion
service /onlineshop on on new http:Listener(9090) {
```

Since there is only one service on the listener, it can be replaced like this.

Reply:
changed in https://github.com/ballerina-platform/module-ballerinax-oracledb/pull/36/commits/669df6defa5946c90f034656ef97f68df60f7d4e